packages/node-cli/src/SdkExecutor.js

```diff
@@ -29,7 +30,6 @@ module.exports = class SdkExecutor {
 		this._sdkPath = sdkPath;
 
 		this._CLISettingsService = new CLISettingsService();
-		this._environmentInformationService = new EnvironmentInformationService();
```

Comment:
The `this._environmentInformationService = new EnvironmentInformationService();` will still be needed by `_checkIfJavaVersionIssue()`

Reply:
Added back.